tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -323,4 +324,90 @@ var _ = Describe("Edit machinepool",
 				Expect(mp.Labels).To(Equal(strings.Join(common.ParseCommaSeparatedStrings(labels), ", ")))
 				Expect(mp.Taints).To(Equal(strings.Join(common.ParseCommaSeparatedStrings(taints), ", ")))
 			})
+
+		It("Validate --worker-mp-labels option for ROSA cluster creation - [id:71329]",
```

Comment:
I would put that in `test_rosacli_cluster.go` file, in Validation features

Reply:
I have shifted the case to `test_rosacli_cluster.go`